services/core-web/src/components/mine/Permit/ViewPermit.tsx

```diff
@@ -60,13 +62,14 @@ const ViewPermit: FC = () => {
   const [pollForStatus, setPollForStatus] = useState(false);
 
   const hasConditions = latestAmendment?.conditions?.length > 0;
-  const isReviewComplete = latestAmendment?.conditions?.every((con) => con.permit_condition_status_code === PERMIT_CONDITION_STATUS_CODE.COM);
-
+  const isReviewComplete = latestAmendment?.conditions?.every(
+    (con) => con.permit_condition_status_code === PERMIT_CONDITION_STATUS_CODE.COM
+  );
 
   const canStartExtraction =
-    ((documents.length > 0 && !permitExtraction?.status) ||
+    ((documents.length > 0 && !permitExtraction?.task_status) ||
       [PermitExtractionStatus.error, PermitExtractionStatus.not_started].includes(
-        permitExtraction?.status
+        permitExtraction?.task_status
```

Comment:
Y'know, I _had_ noticed that something was funky with how it displayed when there was no extraction- was that it? We were using status when the actual property name was task_status? Ahh, to get TS working properly with redux...

Reply:
It was!  I only noticed it though because Typescript showed the error and I followed the thread to find out it was right!  Yay Typescript!